The-Recipe-Manager-App/src/application/CardController.java

```diff
@@ -102,14 +122,8 @@ public class CardController implements Initializable {
     @FXML
     private ImageView recipeRatingDetail;
 
-
-   
-    private Stage stage;
-    private Scene scene;
-
-    
-
-    private String[] colors =  {"DCEDF2", "FFFFF"};
+    @FXML
+    private Button MoreBtnCard;
     
```

Comment:
see if they need private stage and scene here

Reply:
They are not needed because there is only one place where we need them and we already created there.